test/controllers/auth_controller_test.exs

```diff
@@ -3,7 +3,9 @@ defmodule AuthControllerTest do
   alias Constable.{User, UserIdentifier, UserInterest}
 
   @google_authorize_url "https://accounts.google.com/o/oauth2/auth"
-  @oauth_email_address "fake@thoughtbot.com"
+  @permitted_email_domain Application.fetch_env!(:constable, :permitted_email_domain)
+ 
+  def valid_email_address, do: "fake@#{@permitted_email_domain}"
```

Comment:
Great question! There is indeed a reason that this file uses a function over a module attribute. 

If the valid email address were defined using a module attribute, the other modules in this file, `FakeRequestWithAccessToken` and `FakeTokenInfoGoogleStrategy`, don't have access to `@valid_email_address`, as it's defined on `AuthControllerTest`. To still define a valid email address format in one location, the best way around this seems to be to define a function on `AuthControllerTest` and call `AuthControllerTest.valid_email_address` within the other modules.

I'm certainly open to an alternative approach if you have one :)